### PR TITLE
Implement of Flavors & Environment

### PR DESCRIPTION
**Pull request by minikin**
- Implement  Flavors & Environment

**NOTE:** The only PRODUCTION Flavor is functional.
We need to create additional app ids on Apple Dev Portal and Google Play. 
I'm going to do so on #326

┆Issue is synchronized with this [Jira Task](https://highvibe.atlassian.net/browse/HVN1-185) by [Unito](https://www.unito.io/learn-more)

### CODE REVIEW

File: lib/modules/wallet/stores/wallet_store.dart [hunk 1]
```diff
@@ -163,4 +124,43 @@ abstract class WalletStoreBase with Store {
...
+  Future<void> _initialiseFromPrivateKey(String privateKey) async {
+    try {
+      final address = await _addressService.getPublicAddress(privateKey);
+
+      this.address = address.toString();
+      this.privateKey = privateKey;
+
+      await _initialise();
+    } catch (error) {
+      print(error);
```
Comment (AlexVegner) [inline lib/modules/wallet/stores/wallet_store.dart:163]: Avoid to use print(error) use log.e(error) from lib/utils/log.dart

Comment (minikin) [reply lib/modules/wallet/stores/wallet_store.dart:163]: Btw we might need to consider to move `logger` to `dev_dependencies`

File: README.md [hunk 2]
```diff
@@ -5,28 +5,37 @@ HighVibe.Network is a blockchain-based ecosystem designed to incentivize persona
...
 - Flutter & Dart plugins:
   - [Visual Studio Code](https://flutter.dev/docs/get-started/editor?tab=androidstudio)
   - [Android Studio / IntelliJ](https://flutter.dev/docs/get-started/editor?tab=vscode)
 
-## Setup
+## Contributing
+
+### Secrets
+
+1. Get `secrets.dart` file from 1Password team account
```
[the remark this answers is not in the document]
Comment (minikin) [reply README.md:33]: Such an approach means that we won't be able to test apps locally against other environments except for development. Or do you have something else in mind?

File: .vscode/launch.json [hunk 3]
```diff
@@ -5,16 +5,49 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "Debug",
+      "name": "Debug : Dev",
       "request": "launch",
       "type": "dart",
-      "flutterMode": "debug"
+      "program": "${workspaceRoot}/lib/configurations/flavors/main_development.dart",
```
[the remark this answers is not in the document]
Comment (minikin) [reply .vscode/launch.json:11]: I feel like `flavors ` is a better place for `main_xxx.dart` files but I don't have any serious objections about your request :)

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (minikin) [reply README.md]: We have this process automated now. 
1. Secret files are saved as encrypted files in the [secrets folder](https://github.com/HighVibe/hvn/tree/feature/sp-325/secrets) using AES256 algorithm.
2. CI/СD scripts automatically obtain the secret key from GitHub's secrets storage. 
3. To decrypt locally on developers' machines (macOS only at this moment) developers should [follow instruction](https://github.com/HighVibe/hvn/tree/feature/sp-325#secrets).
4. We'll keep `decrypt_secrets.sh` up to date.

**Pull request merged**